code-challenges.test.js

```diff
@@ -52,5 +151,43 @@ const numbersToAdd2 = [0, 7, -8, 12]
 const numbersToAdd3 = []
 // Expected output: []
 
-
 // b) Create the function that makes the test pass.
+
+const addition = (numbers) => {
+    let newArr = []
+    for(let i=1; i<numbers.length; i++)
+    newArr.push(numbers[i] + 2)
+}
+
+// Psuedocode:
+// 1. Declare a function called addition
+// 2. Create a parameter called numbers
+// 3. Create a variable called newArr 
+// 4. Create a for loop to iterate from the second index(index 1) because for the expected output of each array the 0 index remains the same.
+// 5. Create return that'll give the expected output
+
+//--------------Notes question 3----------------- Since there's no pattern in the array I tried to do the addition but unsure how to write for each array. Since there's 3 seperate arrays do i need to write a function for each and how do I incude that in the function. 
+//error message:
+//takes in an array and returns an array of the accumulating sum.
+
+// ReferenceError: array is not defined
+
+// 154 | const addition = (numbers) => {
+// 155 |     let newArr = []
+// > 156 |     for(let i=1; i<array.length; i++)
+//     |                    ^
+// 157 |     newArr.push(array[i] + 2)
+// 158 | }
+// 159 |
+//----------------so after that I switched all the arrays in my function to numbers I got this error:
+
+//Expected: [2, 6, 51, 60]
+// Received: undefined
+
+// 137 |         const numbersToAdd2 = [0, 7, -8, 12]
+// 138 |         const numbersToAdd3 = []
+// > 139 |         expect(addition(numbersToAdd1)).toEqual([2, 6, 51, 60])
+//     |                                         ^
+// 140 |         expect(addition(numbersToAdd2)).toEqual([0, 7, -1, 11])
+// 141 |         expect(addition(numbersToAdd3)).toEqual([])
```

Comment:
Again, great job with your tests!  The error you are getting at the end is that your function's output is not what you told it to expect.  Just means you need to adjust your function.  This one is tough. You don't need to write 3 separate functions. Rather one function that will work on each of the three variables.